package.json

```diff
@@ -43,7 +43,8 @@
         "@types/apollo-upload-client": "^8.1.3",
         "@types/graphql": "^14.2.3",
         "@types/jest": "^24.0.15",
-        "@types/lodash": "^4.14.138",
+        "@types/lodash": "^4.14.154",
+        "@types/nightwatch": "^1.1.3",
```

Comment:
nightwatch should probably not be re-added 😄

Reply:
Done. It gets auto install for whatever reason